.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,50 @@
+# Copyright the Hyperledger Fabric contributors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: Pull Request
+on:
+  push:
+    branches: ["**"]
+  pull_request:
+    branches: ["**"]
+  workflow_dispatch:
+
+env:
+  GOPATH: /opt/go
+  PATH: /opt/go/bin:/bin:/usr/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin
+  GO_VER: 1.18.2
+
+jobs:
+  verify-build:
+    name: Verify Build
+    runs-on: ubuntu-20.04
+    steps:
+      - run: sudo apt-get clean
```

Comment:
For several Ubuntu releases, `apt` has been the preferred command for package management with `apt-get` remaining for backwards compatibility. It might be a good time to switch. The commands are the same so it's just a case of replacing `apt-get` with `apt`.